targets/TARGET_Freescale/TARGET_KLXX/TARGET_KL25Z/device/TOOLCHAIN_GCC_CW_NEWLIB/MKL25Z4.ld

```diff
@@ -0,0 +1,160 @@
+/* Linker script for mbed LPC1768 */
```

Comment:
This folder has been deleted as part of PR #9714

Reply:
Will remove this file 👍

Reply:
@ARMmbed/mbed-os-maintainers - GCC_CW and other variants cleanup was done in previous PR. I will add another PR to remove this file since CI testing is done and it is almost ready to merge. Let me know if that is a problem

Reply:
we are still waiting for few more reviews until Monday, lets remove it here and we restart CI

Reply:
Done